k8s/environments/prod/cluster-00/mi/mi-adf-shir.yaml

```diff
@@ -0,0 +1,25 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: mi-adf-shir-deployment
+  namespace: mi
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.chart-image: glob:prod-*
+spec:
+  releaseName: mi-adf-shir
+  chart:
+    git: git@github.com:hmcts/shared-services-flux
+    path: k8s/release/mi/mi-adf-shir
+    ref: master
+  values:
+    labels:
+      app.kubernetes.io/instance: mi-adf-shir-deployment
+      app.kubernetes.io/name: mi-adf-shir-deployment
+    image: sdshmctspublic.azurecr.io/mi/adf-integration-runtime:prod-09b63f3
+    secretsMountPath: ''
+    environment: "prod"
+    resourceGroup: "mi-prod-rg"
+    subscriptionId: "5ca62022-6aa2-4cee-aaa7-e7536c8d566c"
```

Comment:
Can we drive these via devops variables groups?  I did start creating these for the framework for a future piece of work on the framework:
![image](https://user-images.githubusercontent.com/85565430/129029529-8ed805c4-c2b7-49e4-8251-76c24a0ba7a1.png)

Reply:
This isn't our repo so we can't I don't think. But this will be migrated to kustomize in the future:
https://tools.hmcts.net/jira/browse/RMI-1622
Which will reduce the duplication.